simulator/simulator.py

```diff
@@ -112,6 +112,34 @@ def action(self, actions: list[dict]) -> bool:
         Perform an action.
         """
 
-        print("WARNING: actions are not yet implemented for this Simulator.")
+        for action in actions:
+        # iterate through the list of dictionaries (actions) with action 
+
+            if action["type"] == "flight_level":
+                # if you find this key in any of the action dictionaries execute if statement
+                self._delta_flight_level(action)
+                # execute this function on the action dictionary
+
+        # print("WARNING: actions are not yet implemented for this Simulator.")
```

Comment:
Perhaps uncomment this and add it behind an `else` so it's explicitly printed if an action is sent that is not understood